.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
in https://github.com/hpi-schul-cloud/node-notification-service/pull/273/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R21 the .gitignore file is used, the script lint should be updated in package.json

Reply:
I configured my vscode workspace to use .gitignore as ignorePath and removed .eslintignore.